needrestart

```diff
@@ -660,6 +661,25 @@ if(defined($opt_l)) {
 				print STDERR "$LOGPREF #$pid part of user manager service: uid=$1\n" if($nrconf{verbosity} > 1);
 				push(@{ $sessions{$1}->{'user manager service'}->{ $ptable->{$pid}->{fname} } }, $pid);
 				next;
+			    }
+				if($value =~ m@/machine.slice/machine.qemu(.*).scope@) {
+				for my $cmdlineidx (0 .. $#{$ptable->{$pid}->{cmdline}} ) {
+					if ( ${$ptable->{$pid}->{cmdline}}[$cmdlineidx] eq "-name") {
+						foreach ( split(/,/, ${$ptable->{$pid}->{cmdline}}[$cmdlineidx+1]) ) {
+							if ( index($_, "guest=") == 0 ) {
+								my @namearg = split(/=/, $_, 2);
+								if ($#{namearg} == 1) {
+									print STDERR "$LOGPREF #$pid detected as VM guest '$namearg[1]' in group '$value'\n" if($nrconf{verbosity} > 1);
```

Comment:
This would need to be localized.

Reply:
here it is harder for me to understand where to add the localization.
I aligned to what was there like `print STDERR "$LOGPREF #$pid unexpected cgroup '$value'\n" if($nrconf{verbosity} > 1);` which doesn't use localization either AFAICS.

If you have a hint how you'd want me to add it here, please let me know.

Reply:
The debugging stuff is not to be localized but all normal output. In This case we something like:

```perl
  push(@guests, __x("'{name}' with pid {pid}", name => $namearg[1], pid=>$pid) );
```

(Sorry it looks like I added the review comment at the wrong line number. the push in line 681 also needs a localization).

Reply:
Thanks for explaining that @liske 
I have updated both not yet localized pushed using the __x feature you used everywhere else.

In addition I rebased to the latest master branch.